src/runtime/pkg/sev/README.md

```diff
@@ -1,14 +1,15 @@
 # AMD SEV confidential guest utilities
 
-This package provides utilities for launching AMD SEV confidential guests.
+This package provides utilities for launching AMD SEV/SEV-ES confidential
+guests.
```

Comment:
Is it worth noting which HW generations this package is compatible with?

Reply:
Rome and above?
Naples (for SEV) and Rome (for SEV-ES) and above?

BTW is this the hardware requirements documented anywhere for Confidential Containers?

Reply:
I was thinking about the CPU signatures not including Genoa or newer.

We don't really have a comprehensive listing of hardware requirements.

Reply:
Genoa signatures added.

Reply:
That's an interesting idea. Reminds me a bit of what is done for EC2. It might be more elegant to remove the cpu sig from the TCB entirely rather than report a "wrong" one, but that's a lot more work.